packages/components/src/components/buttons/ButtonGroup/ButtonGroup.tsx

```diff
@@ -14,15 +17,17 @@ const Container = styled.div<{ $variant?: Exclude<ButtonVariant, 'danger'> }>`
         border-radius: 0;
     }
 
-    > :first-child {
+    > ${({ $withTooltips }) =>
+            $withTooltips ? 'div:first-child > div > button' : ':first-child'} {
         border-radius: ${borders.radii.full} 0 0 ${borders.radii.full};
     }
 
-    > :last-child {
+    > ${({ $withTooltips }) => ($withTooltips ? 'div:last-child > div > button' : ':last-child')} {
         border-radius: 0 ${borders.radii.full} ${borders.radii.full} 0;
     }
 
-    > :not(:last-child) {
+    > ${({ $withTooltips }) =>
+            $withTooltips ? 'div:not(:last-child) > div > button' : 'button:not(:last-child)'} {
         position: relative;
```

Comment:
@jvaclavik please, is this ok usage?

Reply:
It's that if you use `IconButton` with prop `label` it will wrap the component in tooltip and then corner radiuses and dividers are completely broken.

It's just for tokens in Figma - [here is the link](https://www.figma.com/design/o6eb4wUMjSu5xMb34423vh/Tokens?node-id=4084-12992&t=KrMgNjbWidsDL3El-0).

Was not really sure how to approach it, was trying with something like button:first-of-type but the problem is with nesting in elements.

withTooltips=true is meant to be for usage with tooltips inside.

Reply:
This is not correct way how to fix it. But it's definitely an issue in our `ButtonGroup` component.

@AdamSchinzel Fix is on the way: https://github.com/trezor/trezor-suite/pull/12646

Reply:
PR is merged: https://github.com/trezor/trezor-suite/pull/12646